lib/internal/event_target.js

```diff
@@ -494,13 +494,13 @@ class EventTarget {
 ObjectDefineProperties(EventTarget.prototype, {
   addEventListener: { enumerable: true },
   removeEventListener: { enumerable: true },
```

Comment:
These should be using `kEnumerableProperty` which is currently available under `internal/webstreams/util`. Worth fixing those in the same PR? Ideally we should move `kEnumerableProperty` to `internal/util` so it can be another PR.

Reply:
That's a good idea, however I think we should move them and replace them with the `kEnumerableProperty` symbol in a different PR, do you suggest we do them in this PR or another?